nexus-sled-agent-shared/src/inventory.rs

```diff
@@ -238,9 +238,6 @@ pub enum OmicronZoneType {
     },
     InternalNtp {
         address: SocketAddrV6,
-        ntp_servers: Vec<String>,
-        dns_servers: Vec<IpAddr>,
-        domain: Option<String>,
```

Comment:
I'm assuming this means that Sled Agent will no longer be _sending_ this information as part of inventory, and that this is okay because we're still doing offline upgrades so Nexus will always be sync'd up?